sw/device/silicon_creator/rom_exts/docs/manifest.md

```diff
@@ -8,457 +8,87 @@ aliases:
   Status: Draft
 </p>
 
-This describes the signed image format for a ROM_EXT image.
-This format is based on the requirements outlined in the [Reference Mask ROM Secure Boot Description][mask-rom-description].
+This document describes the manifest format for boot stage images stored in flash.
+
+OpenTitan secure boot, at a minimum, consists of three boot stages: `ROM`,
+`ROM_EXT`, and the first owner boot stage, e.g. `BL0`. `ROM` is stored in the
+read-only Mask ROM while remaining stages are stored in flash. `ROM` and
+`ROM_EXT` require the manifest described in this document to be at the start of
+`ROM_EXT` and first owner boot stage images so that they can verify the
+integrity and authenticity of the next stage and configure peripherals as
+needed before handing over execution. Use of this manifest for stages following
+the first owner boot stage is optional.
+
+The following table lists the fields of the manifest along with their sizes,
+alignments, and offsets from the start of the manifest in bytes. The last
+column provides the corresponding C data type of each field for illustration
+purposes. All manifest fields are stored little-endian and the total size of
```

Comment:
Is `signature[0]` the least significant or most significant word in the signature? If it is the most significant then we should call that out explicitly as it means the signature field is not encoded as a large little-endian number.

Reply:
It's the least significant word of the signature, i.e. consistent with all other fields and the text you quoted.